base/sysinfo.jl

```diff
@@ -98,7 +98,33 @@ Standard word size on the current machine, in bits.
 """
 const WORD_SIZE = Core.sizeof(Int) * 8
 
-global SC_CLK_TCK::Clong, CPU_NAME::String, JIT::String
+global SC_CLK_TCK::Clong
+
+"""
+    Sys.CPU_NAME::String
+
+A string representing the name of CPU.
+
+# Examples
+```
+julia> Sys.CPU_NAME
+"tigerlake"
+```
```

Comment:
I think it would be clearer to use a sentence here, since you don't need to show how to call it, names like `"tigerlake"` are pretty obscure, and everyone's may be different.  For example:
```
For example, `Sys.CPU_NAME` might equal `"tigerlake"` on an 
[Intel Core "Tiger Lake" CPU](https://en.wikipedia.org/wiki/Tiger_Lake),
or `"apple-m1"` on an [Apple M1 CPU](https://en.wikipedia.org/wiki/Apple_M1).
```
```